grapher/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -58,19 +60,32 @@ interface Item {
     bars: Bar[]
 }
 
+interface PlacedItem extends Item {
+    yPosition: number
+}
+
 interface Bar {
     color: Color
     seriesName: string
     point: StackedPoint<EntityName>
 }
 
-interface TooltipProps {
+interface TooltipProps extends StackedBarChartContext {
     label: string
     bars: Bar[]
-    highlightedSeriesName?: string
-    targetTime?: Time
+    highlightedSeriesName: string | null
```

Comment:
Why switch from `string | undefined` to `string | null`?

Reply:
Ah yes. My rationale here was that with `string | null`, we cannot forget to pass `highlightedSeriesName` by mistake because TS would complain about that.

Reply:
Ah right! In that case could the type be `highlightedSeriesName: string | undefined`? I think that's different from `highlightedSeriesName?: string`. In the 1st case the prop needs to be explicitly passed even if it's `undefined`, in the 2nd you can omit it.

Or at least I think so, I might be wrong. 

Nitpicking here because I think it's better to keep to using one of `undefined` or `null`, and the codebase defaults to `undefined` in most cases.

Reply:
You're right there! Thank you, I thought these two behaved the same way.